enterprise/users.go

```diff
@@ -37,8 +38,11 @@ func (c *UserStore) Delete(ctx context.Context, u *chronograf.User) error {
 }
 
 // Get retrieves a user if name exists.
-func (c *UserStore) Get(ctx context.Context, name string) (*chronograf.User, error) {
-	u, err := c.Ctrl.User(ctx, name)
+func (c *UserStore) Get(ctx context.Context, q chronograf.UserQuery) (*chronograf.User, error) {
```

Comment:
are non-Chronograf Users sharing the same Bolt store? if so, we can't just match by Name as that could be non-unique – we'll need to add a `Scheme` and `Provider` (along the lines of `InfluxDB` and `Influx`) to ensure precision.

seems like all of these `enterprise` methods will need to be augmented to reflect the enhanced User model. i'll make the same note on the `influx` methods.

Reply:
I think these users are stored in InfluxDB or in Plutonium.

Reply:
i see. hence pulling in the `timeSeries`? so do we get a view on the source and then search within that? such that Name is all that's required? and never store any of these users in Bolt? cc @goller